.gitignore

```diff
@@ -1,6 +1,7 @@
 .gradle
 build/
 .cxx
+bin
```

Comment:
How are you producing a `bin` directory?

Reply:
I ran `./gradlew linuxX64DistZip`

Reply:
That's strange, there is no reason why a `bin` directory would be created by this command. All build outputs should be under `build/`.

The only explanation I can think of is unzipping a jvm build, which does contain `bin` and `lib` directories.

Reply:
Ok I look now at the code and is strange yes, I will try to see if I can reproduce in a fresh env this `bin` stuff.